app/models/receipt.rb

```diff
@@ -0,0 +1,4 @@
+class Receipt < ApplicationRecord
+  belongs_to :request
+  enum types: { purchase: 0, rental: 1 }
```

Comment:
like I said above, not sure if this is necessary anymore since we record this in the request